blocks/edit/utils/helpers.js

```diff
@@ -127,6 +127,18 @@ function parseAemError(xError) {
   return xError.replace('[admin] ', '');
 }
 
+export async function getCdnConfig(path) {
+  const [org, site] = path.slice(1).toLowerCase().split('/');
+  const resp = await daFetch(`${AEM_ORIGIN}/config/${org}/sites/${site}.json`);
+  if (!resp.ok) return { error: 'Cannot fetch site config.', status: resp.status };
```

Comment:
error / status is never read and fails silently in `getCdnHref` where it will return `url.href` because preview/prod is not defined.

Should we at least console log/warn here?